SharkCage/CageManager/base64.cpp

```diff
@@ -0,0 +1,98 @@
+#include "base64.h"
+#include "stdafx.h"
+#include <string>
+
+
+static const std::string base64_chars =
+"ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+"abcdefghijklmnopqrstuvwxyz"
+"0123456789+/";
+
+
+static inline bool is_base64(unsigned char c) {
+	return (isalnum(c) || (c == '+') || (c == '/'));
+}
+
+std::string base64_encode(unsigned char const* bytes_to_encode, unsigned int in_len) {
```

Comment:
can you replace the raw pointer with a reference? (if it isn't too much trouble)

Reply:
Hmm I tried to refactor the function but jumped then from problem to problem. I'm, quiet sure that it is possible to adjust it to your suggestion but until now I hadn't the time to dive deeper into the problems so it will took some further time but will refactor it 👍!

Reply:
not sure where you call this function (couldn't find it in this pr?), but the easiest solution should be to just expect a const string reference and then get a `char*` from it and leave the rest of the function like it is: http://cpp.sh/9qpix